### DISCUSSION TITLE
Auto Load and Refresh doesn't work - Console Errors

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#5068] [Q&A]
I have been using the latest stable release, and as everything has been going fine, I have been getting issues related to auto-loading more articles at the bottom of the stream footer and the refresh button.

The loader.gif does not load and the "Load more articles" button appears, which remains idle and loads later articles only when the button is clicked.
The refresh button on clicked doesn't do anything, and my console logs an error, which I have discussed down below.

Once the page is loaded, the following error is logged into my console which i suspect could be the reason behind non auto loading of more articles : 

Uncaught TypeError: Cannot read properties of null (reading 'cloneNode')
    at faviconNbUnread (main.js?1674681831:1787:49)
    at init_normal (main.js?1674681831:1840:2)
    at init_main_beforeDOM (main.js?1674681831:1857:3)
    at main.js?1674681831:1881:1

As for the refresh button, when clicked, the following error is shown in the console :-

Uncaught TypeError: Cannot read properties of null (reading 'querySelector')
    at openNotification (main.js?1674681831:1506:15)
    at req.onload (main.js?1674681831:1467:5)
openNotification @ main.js?1674681831:1506
req.onload @ main.js?1674681831:1467
load (async)
actualize.onclick @ main.js?1674681831:1448

I have only made a few changes to the theme (Origine) (mostly padding, font properties) and some en- language file customizations to ./app/i18n/ 

Any help is appreciated. Thank You 

Edit :- The auto load method works on Microsoft Edge (and no error is logged) but not on Chrome or Firefox

**Comment by Alkarex** [comment]
Hello,
 I cannot reproduce the problem. Could you please:

* Try to disable all FreshRSS extensions
* Try with another computer / browser with no browser extension
* Check for any other error in your browser console / network.

What platform are you running your browser on?
How did you install FreshRSS?

**Reply by ghost** [reply]
Thank you for the suggestions.
I am running my browser on Linux (Elementary OS) and the server on Apache.

I doubt it is the problem with either the internet connection or the browser. Out of the many times i tried running the website, a few times it has worked (load more articles) and has not given any error in my console.

But, can you clarify what the console errors I have posted are about?

**Reply by ghost** [reply]
And for the installation, I installed it manually by downloading repository from GitHub

**Reply by Alkarex** [reply]
The errors you have posted are a sign of something interfering with FreshRSS. More precisely, some missing elements in the page (so either some wrong manual changes, some browser extension, FreshRSS extension, or something else).
Please check again by post above for how you should test.
You can also send me ([by e-mail](https://alexandre.alapetite.fr/cv/)) a test account on your instance so I can test by myself.
